programs/bpf_loader/src/syscalls/mem_ops.rs

```diff
@@ -550,6 +554,41 @@ impl<'a> DoubleEndedIterator for MemoryChunkIterator<'a> {
             }
         };
 
+        let region_is_account;
+
+        let mut account_index = self
+            .account_index
+            .unwrap_or_else(|| self.accounts.len().saturating_sub(1));
+        self.account_index = Some(account_index);
+
+        loop {
```

Comment:
This loop could use some comments, it took me half an hour to understand what
was going on 😅

Reply:
That's true for a lot of runtime code. I'll review and add some comments